scripts/run-api-tests.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+
+:: Go to api directory
+set OLDDIR=%CD%
+cd %~dp0..\api
+
+:: Run API
+call .\.venv\Scripts\activate
+set PYTHONPATH=%~dp0..\
+python3 test-api.py
```

Comment:
Idk if it's only for me but python3 is not a defined in my environment so I used py -3 for the setup bat file